src/breadcrumb-group/item/item.tsx

```diff
@@ -98,7 +98,10 @@ export function BreadcrumbItem<T extends BreadcrumbGroupProps.Item>({
   }
 
   return (
-    <div className={clsx(!isGhost && styles.breadcrumb, isGhost && styles['ghost-breadcrumb'], isLast && styles.last)}>
+    <div
+      className={clsx(!isGhost && styles.breadcrumb, isGhost && styles['ghost-breadcrumb'], isLast && styles.last)}
+      data-testid={item.testId}
```

Comment:
The `isGhost` version should not render test-ids. You are stepping directly into this issue: AWSUI-59666

Reply:
Thanks for pointing this out, addressed this and added a test case for it 👍